server/runtime.go

```diff
@@ -77,28 +77,22 @@ func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxR
 	dnsOpts := req.GetConfig().GetDnsOptions()
 	fmt.Println(dnsOpts)
 
-	// TODO: the unit of cpu here is cores. How to map it into specs.Spec.Linux.Resouces.CPU?
-	cpu := req.GetConfig().GetResources().GetCpu()
-	if cpu != nil {
-		limits := cpu.GetLimits()
-		requests := cpu.GetRequests()
-		fmt.Println(limits)
-		fmt.Println(requests)
-	}
-
-	memory := req.GetConfig().GetResources().GetMemory()
-	if memory != nil {
-		// limits sets specs.Spec.Linux.Resouces.Memory.Limit
-		limits := memory.GetLimits()
-		if limits != 0 {
-			g.SetLinuxResourcesMemoryLimit(uint64(limits))
-		}
+	// set up CPU resources
+	limits := req.GetConfig().GetResources().GetCpu().GetLimits()
```

Comment:
> @runcom , I tested this with a config file which does not set memory field, and it works normally.
> However, I must admit I did not fully understand how go makes this work.

Memory field probably isn't nillable then - we should try this on a field which can be - the fact that the receiver isn't nil doesn't mean the value got isn't as well. Not a protobuf expert here but I guess since this _can_ return `nil`, it might so better check those.